tool/make-distribution.sh

```diff
@@ -125,12 +125,12 @@ rm mxbuild/$PLATFORM/dists/truffleruby-zip.tar
 cat > truffleruby_env <<EOS
 export TRUFFLERUBY_RESILIENT_GEM_HOME=true
 export TRUFFLERUBY_CEXT_ENABLED=$TRUFFLERUBY_CEXT_ENABLED
-export TRUFFLERUBYOPT="$TRUFFLERUBYOPT"
+export TRUFFLERUBYOPT="$TRUFFLERUBYOPT \$TRUFFLERUBYOPT"
```

Comment:
I'm really confused by this. You want to add the literal value "$TRUFFLERUBYOPT" to the environment variable `TRUFFLERUBYOPT`?

Reply:
This part is generating a file (`truffleruby_env`) which is automatically `source`-d by the launcher.
There I want to set TRUFFLERUBYOPT based on values above, but also honor TRUFFLERUBYOPT coming from the environment, so I need a literal `$TRUFFLERUBYOPT` in the file:
`export TRUFFLERUBYOPT="-Xgraal.warn_unless=false $TRUFFLERUBYOPT`

Reply:
Thanks for clarifying.